### ISSUE TITLE
Support `labels` in `tune.confusion_matrix`

### ISSUE THREAD

**Issue by gsganden** [gsganden/model_inspector#56]
`sklearn.metrics.confusion_matrix` accepts a `labels` argument, but `model_inspector.tune.confusion_matrix` does not. This functionality would be useful for cases where we want to see all possible label values even if some of those values are not represented.
